plugins/modules/azure_rm_firewallpolicy.py

```diff
@@ -0,0 +1,437 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2021 Aparna Patil(@aparna-patil)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: azure_rm_firewallpolicy
+
+version_added: "1.10.0"
+
+short_description: Create, delete or update specified firewall policy.
+
+description:
+    - Creates, deletes, or updates given firewall policy in specified resource group.
+
+options:
+    resource_group:
+        description:
+            - Name of the resource group.
+        required: true
+        type: str
+    name:
+        description:
+            - The name of the firewall policy.
+        required: true
+        type: str
+    location:
+        description:
+            - Location for firewall policy. Defaults to location of resource group if not specified.
+        type: str
+    base_policy:
+        description:
+            - The name of the parent firewall policy from which rules are inherited.
+        type: str
+    threat_intel_mode:
+        description:
+            - The operation mode for Threat Intel.
+        default: alert
+        type: str
+        choices:
+            - alert
+            - deny
+            - off
```

Comment:
```suggestion
            - 'off'
```

Reply:
Hi @Fred-sun Made the change,  Thanks for reviewing!